src/views/forms/TradeForm/TradeSizeInputs.tsx

```diff
@@ -186,12 +185,25 @@ export const TradeSizeInputs = () => {
       </$ToggleButton>
     );
 
-    return slotTooltip ? (
-      <WithTooltip slotTooltip={slotTooltip} side="left" align="center">
-        {toggleButton}
-      </WithTooltip>
-    ) : (
-      toggleButton
+    const tooltipContainer = (
```

Comment:
hmm this isn't really a tooltip anymore since we are just rendering it outright. I think its fine tho considering this will be refactored?